index.html

```diff
@@ -2435,13 +2435,20 @@ <h3>Data Schemas</h3>
         <dl>
           <dt><var>credentialSchema</var></dt>
           <dd>
-The value of the <code>credentialSchema</code> <a>property</a> MUST be one or
+            <p>
+The value of the <code>credentialSchema</code> <a>property</a> SHOULD be one or
 more data schemas that provide <a>verifiers</a> with enough information to
-determine if the provided data conforms to the provided schema. Each
+determine if the provided data conforms to the provided schema(s). Each
 <code>credentialSchema</code> MUST specify its <code>type</code> (for example,
-<code>CredentialSchema2022</code>), and an <code>id</code> <a>property</a>
+<code>VerifiableCredentialSchema2023</code>), and an <code>id</code> <a>property</a>
 that MUST be a <a>URL</a> identifying the schema file. The precise contents of
 each data schema is determined by the specific type definition.
+            </p>
+            <p>
+If multiple schemas are present, validity MUST be determined according to the
```

Comment:
```suggestion
If multiple schemas are present, validity is determined according to the
```

If we put a MUST here, we will have to test it in the core data model test suite... it's do-able, but puts a pretty high burden on this feature... and if we don't get enough implementations for this feature, it'll be struck from the specification. Are we sure that we'll have enough implementers that implement two types of JSON Schema checking for this to survive CR?

Reply:
yes, we have the implementers. I'm fine with removing the MUST